event_upload_rotjane.py

```diff
@@ -13,7 +13,7 @@
 
 # settings 
 root_path = 'http://127.0.0.1:8000/rest/'
-authority = ('chow','chow')
+authority = ('nico','Password1')
```

Comment:
Pro-Tip: You can use `git add -p` to go through all diffs one by one and decide whether to include it in the commit or not. That way you can avoid to inadvertently add changes that should stay local only (like this one).